src/components/the-button.ts

```diff
@@ -270,6 +270,11 @@ export default class TheButton extends BotComponent {
                 const tag = entry.tag == "" ? `<@${entry.user}>` : entry.tag;
                 description += `${tag}: ${round(entry.score, 1)}\n`;
             }
+            // Users current score
+            const user = scores.find(entry => entry.user == interaction.user.id);
```

Comment:
Unfortunately since the database query above only grabs the top 15 entries if the user id isn't present in that scores array this won't work (and if it is in the array it'll already be on the scoreboard). The logic here will need to be updated along the lines of (pseudocode) `if(interaction.user.id not in scores) { const user = await this.wheatley.database.button_scoreboard.findOne({ user: interaction.user.id }); ... insert if found ... }`

Reply:
I'm happy to help with this or chat on discord if you have any typescript questions

Reply:
Noted. I've applied changes to address this problem based on our discussions on discord. Hopefully everything looks acceptable! ^^